server/server.js

```diff
@@ -85,6 +86,7 @@ app.use("/api/cart/add", addToCart);
 app.use("/api/cart/remove", removeFromCart);
 app.use("/api/cart/update", updateQuantity);
 app.use("/api/order/confirm", OrderConfirm);
+app.use("/api/admin/", adminOrders);
```

Comment:
adminOrders is for /api/admin/orders, can we have the URL like that here?

Reply:
yes it is like that only in the orders.js file we are having route called /orders. so that I can write multiple routes in same file we have /admin/order/:ordernumber like that in the same file

Reply:
/api/admin seems to be a generic prefix for all the routes we have, we need to point the controllers to specific routes.
This is not a best practice what you have done. Can you google and read about it and see what's the best way to route to controllers?

in this case,
all the requests that matches /api/admin will be directed to adminOrders which is not right

Please find out what is the best way and let me know

Reply:
Can I have individual file for every route like /api/admin/orders one file /api/admin/order/:orderId one file like that? I guess other's also doing same. I tried googling many are having controllers and routes folder in our case we don't have controllers folder na? so can I have individual file for each route?

Reply:
routes folder is like the controllers for us, both are same

having individual file for every route is not good i think